tools/repl.md

```diff
@@ -50,7 +50,7 @@ into the REPL. This is useful for importing some code you commonly use in the
 REPL, or modifying the runtime in some way:
 
 ```
-$ deno repl --allow-net --eval 'import { assert } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts"'
+$ deno repl --allow-net --eval 'import { assert } from "https://deno.land/std@0.178.0/testing/asserts.ts"'
```

Comment:
Why fix version to `0.178.0`?

Reply:
Here's the few lines for context, as it is now:
```
$ deno repl --allow-net --eval 'import { assert } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts"'
Deno 1.31.0
exit using ctrl+d or close()
> assert(true)
undefined
> assert(false)
Uncaught AssertionError
    at assert (https://deno.land/std@0.178.0/testing/asserts.ts:141:11)
    at <anonymous>:2:1
```

Notice the std@0.178.0 in the output section. Running the command with any other version than 0.178.0, the AssertionError location will of course be different than the fixed 0.178.0 in the docs.

It's a minor thing, I'm happy just to update to 0.197.0:

```
$ deno repl --allow-net --eval 'import { assert } from "https://deno.land/std@$STD_VERSION/assert/mod.ts"'
Deno 1.36.0
exit using ctrl+d, ctrl+c, or close()
> assert(true)
undefined
> assert(false)
Uncaught AssertionError
    at assert (https://deno.land/std@0.197.0/assert/assert.ts:7:11)
    at <anonymous>:2:1
```

With this change, at least the testing/asserts.ts path is replaced by assert/assert.ts in the output.

Reply:
PR is updated with suggestion (update output to 0.197.0 and use $STD_VERSION for `deno repl` command)